packages/components/pages/f-account-info/src/components/Notifications.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <div>
+        <f-alert
+            v-if="showSaveErrorAlert"
```

Comment:
Is it possible for both alert states to appear at the same time?

Reply:
No